packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -64,19 +69,47 @@ class _GestureArena {
     assert(isOpen);
     members.add(member);
   }
+
+  @override
+  String toString() {
+    final StringBuffer buffer = new StringBuffer();
+    if (members.isEmpty) {
+      buffer.write('<empty>');
+    } else {
+      buffer.write(members.map<String>((GestureArenaMember member) {
+        if (member == eagerWinner)
+          return '$member (eager winner)';
+        return '$member';
+      }).join(', '));
+    }
+    if (isOpen)
```

Comment:
Makes sense. I'll add a PropertyDescriptionMixin to make this case convenient and refactor this code and Decorator